backend/src/main/java/gov/cdc/usds/simplereport/config/exceptions/MisconfiguredApplicationException.java

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.usds.simplereport.config.exceptions;
+
+/**
+ * Exception to throw when the application is misconfigured; 
+ * should be thrown on startup.
+ */
+public class MisconfiguredApplicationException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    public MisconfiguredApplicationException(String message) {
+        super(message);
+    }
```

Comment:
```suggestion
    public MisconfiguredApplicationException(String message, Throwable t) {
        super(message, t);
    }
```

Reply:
going to make a second constructor just for this, since plausibly not every instance of this Exception will be caused by an underlying Exception, but i like this!

Reply:
That is a better idea, yeah. 👍🏻